### ISSUE TITLE
Reset password error should point to AS instance or anaconda.org.

### ISSUE THREAD

**Issue by bkreider** [Anaconda-Server/anaconda-client#205]
This error message is very confusing when using airgap AS.  It looks like the user setup their config wrong.

https://github.com/Anaconda-Server/anaconda-client/blob/develop/binstar_client/commands/login.py#L81
```
msg = ('Sorry. Please try again '
               '(go to https://anaconda.org/account/forgot_password '
               'to reset your password)')
```

**Comment by PeterDSteinberg** [comment]
This raises a more general issue that we should go through this repository and find where anaconda.org is hard-coded, then replace with AS instance.  For example, it also shows up in the output of anaconda-build when AS instance is localhost:
```
$ anaconda-build submit --channel dev --queue psteinberg/abc ./
...
Created archive (17 files); Uploading to binstar
 uploaded 6 of 6Kb: 100.00% ETA: 0.0 minutes

To view this build go to http://anaconda.org/psteinberg/myorg.package1/builds/matrix/5
```

**Comment by PeterDSteinberg** [comment]
Fixed by #208

**Comment by stephenakearns** [comment]
+1

On Wed, Sep 2, 2015 at 1:45 PM, Peter Steinberg <notifications@github.com>
wrote:

> This raises a more general issue that we should go through this repository
> and find where anaconda.org is hard-coded, then replace with AS instance.
> For example, it also shows up in the output of anaconda-build when AS
> instance is localhost:
>
> $ anaconda-build submit --channel dev --queue psteinberg/abc ./
> ...
> Created archive (17 files); Uploading to binstar
>  uploaded 6 of 6Kb: 100.00% ETA: 0.0 minutes
>
> To view this build go to http://anaconda.org/psteinberg/myorg.package1/builds/matrix/5
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Anaconda-Server/anaconda-client/issues/205#issuecomment-137206464>
> .
>



-- 
*Stephen Kearns*
*Product Manager*
